locations/spiders/panera_bread.py

```diff
@@ -0,0 +1,107 @@
+import json
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+DAYS = {'MONDAY': 'Mo', 'TUESDAY': 'Tu',
+        'WEDNESDAY': 'We', 'THURSDAY': 'Th',
+        'FRIDAY': 'Fr', 'SATURDAY': 'Sa',
+        'SUNDAY': 'Su'}
+
+
+class PaneraBread(scrapy.Spider):
+
+    name = 'panera'
+    download_delay = 1.5
+    allowed_domains = ["panerabread.com"]
+    start_urls = (
+        'https://locations.panerabread.com/index.html',
+    )
+
+    def store_hours(self, day_hours):
+        day_groups = []
+        this_day_group = {}
+
+        for day_hour in day_hours:
+            hours = ''
+            day, intervals = day_hour['day'], day_hour['intervals']
+            short_day = DAYS[day]
```

Comment:
Instead of doing this lookup you can do `day.title()[:2]`.

Reply:
Thank you @iandees  :)